modules/apps/foundation/portal/portal-messaging-api/bnd.bnd

```diff
@@ -0,0 +1,6 @@
+Bundle-Name: Liferay Messaging API
+Bundle-SymbolicName: com.liferay.messaging.api
+Bundle-Version: 1.0.0
+Export-Package:\
+	com.liferay.messaging,\
```

Comment:
here you have to be careful that you remove the continuation at the end of the line.